wiki/Tournaments/oMMFF/1/en.md

```diff
@@ -66,8 +66,6 @@ The **osu! Multi Mode French Fiesta** (***oMMFF***) is a French double-eliminati
 
 ## Prizes
 
-In every tournament, people conquering a place on the podium are eligible to receive unique prizes created specifically for osu! Multi Mode French Fiesta winners. It is possible that these items change with every installment of the osu! Multi Mode French Fiesta.
```

Comment:
Was there a reason to remove this?

Reply:
Was a conversation between Joehuu and clayton to remove it, quoting @cl8n 
> (applies to quite a few other tournament articles) not sure if this is even useful information, reads like fluff bc it's kind of obvious that the prizes aren't exactly the same as other tournaments in the series (since they are listed right here)